### PR TITLE
first draft of slide templates

### LINKED ISSUE #12: Migrate / transition to a Greenwood Theme Pack
## Type of Change
- [x] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [ ] Bug
- [ ] Other (please clarify below)

## Summary
This would effectively allow this repo to additionally be consumed as a package from npm in a Greenwood project using the (soon to be released) Theme Pack capabity.

## Details
This would require such tasks like
1. Updating _README.md_ to provide broad array of usage options
  - "clone" and go with git
  - degit
  - npm / Greenwood
  - copy / paste!
1. Updating package.json and creating a prerelease script to correct prepare templates and pages

### PR DESCRIPTION
**Pull request by aholtzman**
## Related Issue
resolves issue #6 when complete

## Summary of Changes
added 4 templates and modified another to get started.

### CODE REVIEW

**Comment by thescientist13** [comment]
Awesome, thank you for getting this up!  

Did a quick scan of the deploy preview and looks nice and clean!  Will go through it a bit more a little later on today.   👀

**Comment by aholtzman** [comment]
It is meant as a starting place to have sufficient usable slide templates for your presentation. Anything else you want added or changed I am happy to work on.

**Review by thescientist13 — COMMENTED** [review]
This is looking great!  🏆 

I had an idea and was curious on your thoughts.  I bumped the `font-weight` for the `h1` selector in _main.css_ and I think it helped break things up a bit.  I also bumped up the `font-size` in the title template to compensate.

Screen Shot 2021-07-27 at 12 30 05 PM
Screen Shot 2021-07-27 at 12 30 15 PM

**Comment by aholtzman** [comment]
Those are positive changes in my opinion.

File: src/pages/slides/7.md [hunk 1]
```diff
@@ -1 +1,13 @@
...
+
+# Thank You!
+
+@twitter_name
+
+facebook
+
+email@example.com 
+
+![kitten](https://chuckanddons.com/media/wysiwyg/kitten_blog.jpg)
```
Comment (thescientist13) [inline src/pages/slides/7.md:13]: 🐱

**Review by thescientist13 — APPROVED** [review]
Thanks @aholtzman , this is super helpful!  🙏 😊  

I'm going to merge what's here so far, and keep the issue open for further contributions and opportunities.  🍻

**Pull request merged**